src/main/java/Tyler.java

```diff
@@ -0,0 +1,189 @@
+                      Task removeTask = taskList.get(taskNumber - 1);
+                      taskList.remove(taskNumber - 1);
+                      curr--;
+                      System.out.println("    Noted! I've deleted this task:");
+                      System.out.println("      " + removeTask.toString());
+                      System.out.println("    Now you have " + curr + " tasks in the list");
+
+
+                  } else {
+                      throw new UndefinedActionException();
+                  }
+                  System.out.println("    --------------------------------------------------");
+              } catch (MarkNothingException | UndefinedActionException | EmptyNameException e) {
+                  System.out.println("    " + e.getMessage());
+              }
+          }
+    }
+
+    public static void list() {
+        for(int i = 1; i < taskList.size() + 1; i++) {
+            if (taskList.get(i - 1) == null) {
+                break;
+            }
+            String task = taskList.get(i - 1).toString();
+            System.out.println("    " + i + ". " + task);
+        }
+    }
+
+    public static void greet() {
+        System.out.println("    Hello from Tyler");
+        System.out.println("    What can I do for you?");
+        System.out.println("    list, todo, deadline, event, mark, unmark, bye");
+        System.out.println("    --------------------------------------------------");
+    }
+
+    public static void loadTask() throws IOException {
+        try {
+            File file = new File(FILE_PATH);
+            File parentD = file.getParentFile();
+            if (!parentD.exists()) {
+                parentD.mkdirs();
+            }
+            Scanner load = new Scanner(file);
+            while (load.hasNextLine()) {
+                String line = load.nextLine();
+                String[] input = line.split(" \\| ");
+                String type = input[0];
+                boolean isDone = input[1].equals("1");
+                if (type.equals("T")) {
+                    taskList.add(new Todo(input[2], isDone));
+                } else if (type.equals("D")) {
+                    taskList.add(new Deadline(input[2], LocalDateTime.parse(input[3], OUTPUT_DATE_FORMAT), isDone));
+                } else if (type.equals("E")) {
+                    LocalDateTime start = LocalDateTime.parse(input[3], OUTPUT_DATE_FORMAT);
+                    LocalDateTime end = LocalDateTime.parse(input[4], OUTPUT_DATE_FORMAT);
+                    taskList.add(new Event(input[2], start, end, isDone));
+                }
+                curr++;
+            }
+        } catch (FileNotFoundException e) {
+            File file = new File(FILE_PATH);
+            file.createNewFile();
+        }
+    }
+
+    public static String toFileString(Task task) {
+        String name = task.name;
+        boolean isDone = task.isDone;
+        int isDoned = isDone ? 1 : 0;
```

Comment:
Maybe the "is" variable naming should be reserved for boolean values whereas in this case it is an integer

Reply:
Good suggestion. I will change it later.